roles/openshift_control_plane/tasks/main.yml

```diff
@@ -129,6 +129,17 @@
   # copy to this location to bypass initial bootstrap request
   - /etc/origin/node/node.kubeconfig
 
+- name: Check that origin image is present
```

Comment:
We could move these bits to the start of this file, make the task async, and then poll for completion later to save lots of time.

Reply:
Good idea, implemented that